run/single_native.py

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/python3
+import os
+import re
+import sys
+from subprocess import check_output, STDOUT, CalledProcessError
+
+COVID_SIM_BUILD_PATH = "/build/experiment/src/CovidSim"
+PROJ_ROOT = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
+SCRIPT_DIR = "{}/third-party/covid-sim/data".format(PROJ_ROOT)
+DATA_DIR = SCRIPT_DIR
+NOT_COUNT_SETUP = True
+
+
+def run_single_python(country, num_omp_threads, debug):
+    """
+    Run a single run_sample execution
+    """
+    print("CovidSim single execution w/ parameters:")
+    print("\t- BUILD_PATH: {}".format(COVID_SIM_BUILD_PATH))
+    print("\t- SCRIPT_DIR: {}".format(SCRIPT_DIR))
+    print("\t- Country: {}".format(country))
+    print("\t- Num. OMP Threads: {}".format(num_omp_threads))
+
+    # Prepare cmd
+    _cmd = [
+        "{}/run_sample.py".format(SCRIPT_DIR),
+        country,
+        "--covidsim {}".format(COVID_SIM_BUILD_PATH),
+        "--threads {}".format(num_omp_threads),
+        "--outputdir /tmp/",
+    ]
+    cmd = " ".join(_cmd)
+    print(cmd)
+
+    # Run and check_output
+    _out = check_output(cmd, shell=True, stderr=STDOUT).decode("utf-8")
+    if debug:
+        print(_out)
+    exec_times = re.findall("Model ran in ([0-9.]*) seconds", _out)
+    if COUNT_SETUP:
+        exec_times += re.findall("Model setup in ([0-9.]*) seconds", _out)
+    total_time = sum([float(val) for val in exec_times])
+    print(total_time)
+    print("------------------------------------------")
+
+
+def clean_duplicates(country):
+    for f in [
+        "/tmp/{}_pop_density.bin".format(country),
+        "/tmp/Network_{}_T1_R3.0.bin".format(country),
+    ]:
+        try:
+            os.remove(f)
+        except FileNotFoundError:
+            continue
+
+
+"""
+Example Command Line script for a no intervention simulation
+    /build/experiment/src/CovidSim \
+        /c:1 \
+        /A:/code/experiment-covid/third-party/covid-sim/data/admin_units/Virgin_Islands_US_admin.txt \
+        /PP:/code/experiment-covid/third-party/covid-sim/data/param_files/preUK_R0=2.0.txt \
+        /P:/code/experiment-covid/third-party/covid-sim/data/param_files/p_NoInt.txt \
+        /O:/tmp/Virgin_Islands_US_NoInt_R0=3.0 \
+        /D:/tmp/wpop_us_terr.txt \
+        /M:/tmp/Virgin_Islands_US_pop_density.bin \
+        /S:/tmp/Network_Virgin_Islands_US_T1_R3.0.bin \
+        /R:1.5 98798150 729101 17389101 4797132
+"""
```

Comment:
Is this documentation or a comment relating to some code? If it's docs you could put it in the README